docs/user/publication/index.md

```diff
@@ -0,0 +1,87 @@
+
+## How Can I Publish My Content?
+
+To publish a content in the Library, open the content folder you wish to publish in the builder and click on the publication page button.
+
+![How to open the publication page](open_publish_settings.jpeg)
+
+Now that the publication page is open, follow the rest of the documentation.
+
+:::warning
+Only folders can be published in the Graasp Library. It's also important to note that all the content inside a folder will be published too.
+:::
+
+### Publication Page
+
+The publication page allows you to edit your current item (if the item has children, you need to open the publication page for each item) and visualize how it will be rendered in the library.
+
+On a large screen, such as a computer, the publication page is divided into two parts. You can find the preview of the publication on the left and the publication's status on the right.
+
+![Example of the publication page](publication_page.jpeg)
+
+The preview allows you to edit the item's data, such as its title, description, thumbnail, or tags, for example, and these are displayed similarly to how they appear in the Library.
+
+On the right side, the publication status section lets you know if the item is published. This section is also where you can trigger the publication process.
+
+### Why Are There Warnings?
+
+If you open the publication page of an unedited item, you will see many warnings next to the item's data (like the description, the thumbnail, etc.).
+
+![An empty item displays many warnings](empty_publish_settings_page.jpeg)
+
+These warnings help you improve your publication to increase its visibility in the library. To get more information, hover the mouse over the warning.
+
+![Get more information by hovering over a warning](warning_tooltip.png)
+
+When you have completed all the fields, you should no longer see any warnings. Your content is then ready to be published.
+
+:::note
+You can publish your content even if there are still some warnings, but we encourage you to try to complete all the fields (in all the elements of your content, not just in the root folder) to maximise visibility in the Graasp Library, and to increase the chances that other people will use your content.
+:::
+
+### Publish Your Content
+
+When you are ready, click the `Publish` button to trigger the publication process. It is important to understand that your folder and all its children will be visible to anyone at the end of this process. If your folder is private, the following modal will be displayed, asking you to change its visibility to public. This change will only happen at the end of the process.
+
+![Change the item's visibility](make_it_public.jpeg)
+
+At the moment you trigger a publication, it will have to be validated and then published. This may take a few moments depending on the number of elements. You'll see a "Pending" status while your content is being validated. If you don't see an update soon, try refreshing the page.
```

Comment:
```suggestion
Once you click on the `Publish` button, the collection will first go through an approval process. This may take a few moments depending on the number of elements. You'll see a "Pending" status while your content is being validated. If you don't see an update soon, try refreshing the page.
```

Reply:
Maybe "If you don't see an update soon, try refreshing the page." should go in a warning paragraph.